plugins/worker_tasks/server/rest.py

```diff
@@ -64,7 +64,9 @@ def _validateTask(self, item):
             ioSpec['format'] = ioSpec.get('format', 'none')
             ioSpec['type'] = ioSpec.get('type', 'none')
 
-        return spec
+        handler = item.get('meta', {}).get('workerTaskHandler') or 'worker_handler'
```

Comment:
@zachmullen would it be possible to move this up earlier in the execution and then trigger a handler specific event (e.g. worker_task.{{ handler }}.validate)?  Currently the _validateTask method puts some requirements on input/output types,  mode etc that might not be applicable in all cases (as an extreme example, I could see them message being in some binary format).